.github/workflow/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
```

Comment:
- [ ] Based **on the recommendation of the previous reviewer,** it would be beneficial to **incorporate linter tests into your project**. Currently, **it appears that the issue lies in the name of your folder, as it is labeled `workflow` instead of `workflows`.** By making this adjustment, **you will enable the linter checks to run smoothly in your pull request**, resolving the problem. Thank you for considering this suggestion to improve the functionality and quality of your project.🙋‍♂️

  ![image](https://github.com/YungKingJayy/Hello-Microverse/assets/107952724/9a8a2d05-55c1-440d-8410-53a69aac7572)
  
  **_Example of linter tests running inside a Pull Request:_**
  ![image](https://github.com/YungKingJayy/Hello-Microverse/assets/107952724/4db50d53-2fca-4b6b-9d55-3baee09bde24)